tests/contrib/kafka/test_kafka.py

```diff
@@ -325,6 +327,7 @@ def test_commit_with_consume_with_multiple_messages(producer, consumer, kafka_to
 
 @pytest.mark.snapshot(ignores=SNAPSHOT_IGNORES)
 @pytest.mark.parametrize("should_filter_empty_polls", [False])
+@pytest.mark.skip(reason="FIXME: This test requires the initialization of a new tracer. This is not supported")
```

Comment:
I assume this wasn't super easy to change to using a single tracer. Let's make sure we actually schedule this fix.

Reply:
Yup, I spent an hour on this and couldn't get the test to pass. I'll take another crack at it later today